### ISSUE TITLE
Obfuscate credentials from the logging-operator debug logs

### ISSUE THREAD

**Issue by aslafy-z** [kube-logging/logging-operator#1026]
**Is your feature request related to a problem? Please describe.**
I don't want credentials to be leaked in logs.

**Describe the solution you'd like**
Remove credentials from logging-operator debug flow configuration logs.

**Describe alternatives you've considered**
- Disabling debug logging, not ideal as I want them to stay enabled as it can help understanding what happens.

**Additional context**

Example logs

```
DEBUG   controllers.Logging     flow configuration      {"logging": "/logging-operator-logging", "config": "<source>\n  @type forward\n  @id main_forward\n  bind 0.0.0.0\n  port 24240....."}
```

**Comment by pepov** [comment]
Ideally the renderer should be able to render without secrets, but as a short term solution we can raise the log level to a larger one to avoid the full config output by default.

**Comment by pepov** [comment]
this is fixed in #1700
